ci-operator/templates/cluster-launch-e2e-azure.yaml

```diff
@@ -0,0 +1,189 @@
+# This template is being used for testing OpenShift on Azure. 
+# At the moment it cannot be reused to launch cluster for Origin tests
+kind: Template
+apiVersion: template.openshift.io/v1
+
+parameters:
+- name: JOB_NAME_SAFE
+  required: true
+- name: JOB_NAME_HASH
+  required: true
+- name: NAMESPACE
+  required: true
+- name: CLUSTER_TYPE
+  value: "azure"
+  required: true
+
+objects:
+
+# We want the cluster to be able to access these images
+- kind: RoleBinding
+  apiVersion: authorization.openshift.io/v1
+  metadata:
+    name: ${JOB_NAME_SAFE}-image-puller
+    namespace: ${NAMESPACE}
+  roleRef:
+    name: system:image-puller
+  subjects:
+  - kind: SystemGroup
+    name: system:unauthenticated
+
+# The e2e pod spins up a cluster, runs e2e tests, and then cleans up the cluster.
+- kind: Pod
+  apiVersion: v1
+  metadata:
+    name: ${JOB_NAME_SAFE}
+    namespace: ${NAMESPACE}
+    annotations:
+      # we want to gather the teardown logs no matter what
+      ci-operator.openshift.io/wait-for-container-artifacts: teardown
+  spec:
+    restartPolicy: Never
+    activeDeadlineSeconds: 10800
+    terminationGracePeriodSeconds: 600
+    volumes:
+    - name: shared-tmp
+      emptyDir: {}
+    - name: shared-data
+      emptyDir: {}
+    - name: cluster-secrets-azure
+      secret:
+        secretName: e2e-azure-secret
+
+    containers:
+
+    # Once admin.kubeconfig exists, executes shared tests
+    - name: test
+      image: registry.svc.ci.openshift.org/${NAMESPACE}/pipeline:bin
```

Comment:
You can use `LOCAL_IMAGE_BIN` (see `ci-operator`'s usage message).

Reply:
This one good! Thanks